eng/packaging.targets

```diff
@@ -26,4 +26,16 @@
                       '$(_excludeCompile)' == 'true' and
                       '%(Dependency.Identity)' != '_._'" />
   </Target>
+
+  <PropertyGroup>
+    <PkgProjPath>$(MSBuildProjectDirectory)\..\pkg\$(MSBuildProjectName).pkgproj</PkgProjPath>
+    <!-- Point to our cusotm pack target which builds the pkgproj instead. -->
+    <PackDependsOn Condition="'$(IsSourceProject)' == 'true' and
+                              Exists('$(PkgProjPath)')">_BuildPkgProj</PackDependsOn>
```

Comment:
Cool.  What does Pack target do for our csprojs. Is it a noop?

Reply:
Yeah because `IsPackable` is currently set to false.